addons/gut/test.gd

```diff
@@ -2501,7 +2501,7 @@ func get_summary_text():
 
 ## Create a Double of [param thing].  [param thing] should be a Class, script,
 ## or scene.  See [wiki]Doubles[/wiki]
-func double(thing, double_strat=null, not_used_anymore=null):
+func double(thing, double_strat=null, not_used_anymore=null) -> Variant:
```

Comment:
This can be `Object` I think.  Unless you can't return `null` for `Object`, then it has to remain `Variant`.  I always forget where I can/can't use `null` when something has a type.  I think primitives are the only thing that can't be `null`.